rasa/core/test.py

```diff
@@ -702,25 +702,18 @@ async def _collect_story_predictions(
             success.append(predicted_tracker)
 
     logger.info("Finished collecting predictions.")
-    with warnings.catch_warnings():
-        from sklearn.exceptions import UndefinedMetricWarning
-
-        warnings.simplefilter("ignore", UndefinedMetricWarning)
-        report, precision, f1, accuracy = get_evaluation_metrics(
-            [1] * len(completed_trackers), correct_dialogues
-        )
 
     in_training_data_fraction = _in_training_data_fraction(action_list)
 
+    if len(correct_dialogues):
+        accuracy = accuracy_score([1] * len(correct_dialogues), correct_dialogues)
+    else:
+        accuracy = 0
+
     _log_evaluation_table(
-        [1] * len(completed_trackers),
+        [1] * len(correct_dialogues),
```

Comment:
I'm a bit lost on why this parameter changed. Was it wrong before? It doesn't look like the logic that uses it changed

Reply:
This was unintentional but yes it does not affect the logic at all. I had deleted the use of the whole _log_evaluation_table method here previously. When I put it back in, I just needed a list of 1s with length equal to the number of test stories and I had just used correct_dialogues above so I used that list (I lexically entrained myself :)). Will change it back!

Reply:
Thank you!